tests/fixtures.py

```diff
@@ -39,9 +39,11 @@ def set_backtrace():
 
 @pytest.fixture(scope="session")
 def test_base_dir():
-    d = os.getenv("TEST_DIR", "/tmp")
+    tempfile.tempdir
+    a = tempfile.gettempdir()
+    d = os.getenv("TEST_DIR", a)
 
-    directory = tempfile.mkdtemp(prefix="revaultd-tests-", dir=d)
+    directory = tempfile.mkdtemp(prefix="revaultd-", dir=d)
```

Comment:
I don't think there is a need for removing the `tests-` part anymore

Reply:
Yes, It's needed as I tried keeping it but again got the same error of revaultd crashing.